src/themes/themes.css

```diff
@@ -0,0 +1,115 @@
+/* A basic color theme and associated dark theme inspired by
+   https://material.io/design/color/dark-theme.html.
+
+ Can be imported via:
+
+    @import '~elements-sk/themes/themes';
+*/
+
+@import url(./color-palette.css);
+
+/* Define Color Palette */
+html {
+  /* See https://material.io/design/color/dark-theme.html for how to use each
+     of the below. */
+  --primary: var(--blue-800);
+  --on-primary: var(--white);
+  --primary-variant: var(--blue-600);
+  --on-primary-variant: var(--white);
+  --secondary: var(--deep-orange-800);
+  --on-secondary: var(--white);
+  --background: var(--grey-50);
+  --on-background: var(--black);
+  --surface: var(--white);
+  --on-surface: var(--black);
+  --surface-1dp: var(--grey-100);
+  --surface-2dp: var(--grey-300);
+  --disabled: var(--grey-100);
+  --on-disabled: var(--grey-700);
+  --error: var(--red-900);
+  --on-error: var(--white);
+  --transparent-overlay: rgba(0, 0, 0, 0.5);
+}
+
+/* Class to override colors with dark theme */
+.darkmode {
+  --primary: var(--purple-200);
```

Comment:
IIUC the dark theme colors below are supposed to be copied verbatim from Material Design's baseline dark theme (see [here](https://material.io/design/color/dark-theme.html#ui-application), under heading "The dark theme baseline palette"). However there appear to be some differences.

For example, the primary color here is`--purple-200`, which is defined on `color-palette.css` as `--purple-200: #ce93d8`. But on Material Design's dark theme, the primary color is defined as `#BB86FC`, as seen [here](https://lh3.googleusercontent.com/2tz16tRWvWsNwxg22BmLGDpZ7Pp3SqAg6Zr10WUCbjeEJHFSBzkbb4gwkjIQ5m4rfYoSntR412o7bC7sr8Xi5-gpiciNiQyjssfNWQ=w1064-v0).

If these differences are intentional, it would be useful to include a comment with the criteria used.

Reply:
Added note explaining these match the "names". The values don't seem to line up as you note, I will chalk that up to "some documentation being out of date".